environments/_authorized_keys/pc.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC2AQShgSIH7akDc3F2fdSqUVfA2IUC7jTwEou85eZrMaxi7BKrZuEL5V3RxXg/oDDkmhMtW0r3Ky0yccJQmNy9iTM8Ma8a3LshU1SF2PSU6bGSHiSNcFyIdhQDIk342KeFJQeH+KGAv7t9yP4n/E9rQl3OiIZoT/1ITqQJ6s8aQDVnoNQ4A/Xofa0JnvZzmzX8uID2V6cR0NOXvA2j/M8AS1fBOF7t61LzJvupANYTesX4Kbw6G3ipOmb3Qt/xSnVAywvPl9etFEKYBiuA7MAIycJ9I7A6iy4QRAnX4v5dVjgb5jsaCShnnK7MRqDXvMhVucKqgg6nINABOL5B/LG5 pc@pc-Latitude-7490
```

Comment:
@dannyroberts Is there a way these can be moved to an echis specific directory?

Reply:
Last year we had tried to organize it so that each organization (us, echis, etc.) had their own environments repo, but that we never finished that. So for the time being, unfortunately there isn't